www/front_src/src/Resources/Graph/Performance/Graph/TimeShiftZones/Icon.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { equals, not } from 'ramda';
+
+import { makeStyles } from '@material-ui/core';
+
+import { useMemoComponent } from '@centreon/ui';
+
+import { TimeShiftDirection, useTimeShiftContext } from '.';
+
+export const timeShiftIconSize = 20;
+
+interface Props {
+  xIcon: number;
+  Icon: (props) => JSX.Element;
+  direction: TimeShiftDirection;
+  directionHovered: TimeShiftDirection | null;
+  hoverDirection: (direction: TimeShiftDirection | null) => () => void;
+  ariaLabel: string;
+}
+
+const useStyles = makeStyles({
+  icon: {
+    cursor: 'pointer',
+  },
+});
+
+const TimeShiftIcon = ({
+  xIcon,
+  Icon,
+  direction,
+  directionHovered,
+  ariaLabel,
+  hoverDirection,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const {
+    graphHeight,
+    marginTop,
+    shiftTime,
+    sendingGetGraphDataRequest,
+  } = useTimeShiftContext();
+
+  const translateWithIcon = () =>
```

Comment:
```suggestion
  const attemptToShiftTime = () =>
```

Not sure about this one, but we should find another name